.github/workflows/main.yaml

```diff
@@ -183,14 +183,19 @@ jobs:
 
     - name: Checkout repository
       uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
Is fetching full history required? Could you comment on the motivation for this change?

Reply:
That's for catching that. I wanted to see if a branch name is available in a PR build, hence include it. I have removed it now since it's not needed.